diff --git a/kernel/Kbuild b/kernel/Kbuild
index 4aad0683..f17b4eb2 100644
--- a/kernel/Kbuild
+++ b/kernel/Kbuild
@@ -171,12 +171,12 @@ ifdef CONFIG_KSU_TRACEPOINT_HOOK
   endif
   ifeq ($(CONFIG_KSU),y)
     ifneq ($(KERNEL_TYPE), GKI 2.0)
-      $(info -- $(REPO_NAME) TP Hooks are only supported on GKI 2.0 kernels.)
-      $(info -- Current Kernel Type: $(KERNEL_TYPE))
-      $(info -- If you really want to continue, go to Kbuild and comment out this warning.)
-      $(info -- Usually, this error occurs because you forgot to enable some kernel configurations.)
-      $(info -- Read: https://resukisu.org/guide/manual-integrate.html)
-      $(error TP hooks are incompatible with Non-GKI/GKI 1.0 kernels.)
+      #$(info -- $(REPO_NAME) TP Hooks are only supported on GKI 2.0 kernels.)
+      #$(info -- Current Kernel Type: $(KERNEL_TYPE))
+      #$(info -- If you really want to continue, go to Kbuild and comment out this warning.)
+      #$(info -- Usually, this error occurs because you forgot to enable some kernel configurations.)
+      ##$(info -- Read: https://resukisu.org/guide/manual-integrate.html)
+      #$(error TP hooks are incompatible with Non-GKI/GKI 1.0 kernels.)
     endif
   endif
 
